xbmc/cores/VideoPlayer/Process/ProcessInfo.cpp

```diff
@@ -528,9 +529,20 @@ float CProcessInfo::GetNewTempo()
   return m_newTempo;
 }
 
+float CProcessInfo::MinTempoPlatform()
+{
+  return 0.75f;
+}
+
+float CProcessInfo::MaxTempoPlatform()
+{
+  return 1.55f;
+}
+
 bool CProcessInfo::IsTempoAllowed(float tempo)
 {
-  if (tempo > 0.75 && tempo < 1.55)
+  if (tempo > MinTempoPlatform() &&
+      (tempo < MaxTempoPlatform() || tempo < g_advancedSettings.m_maxTempo))
```

Comment:
Is it wanted behaviour that advanced setting can only raise the allowed tempo?

Reply:
Yes